package/MDAnalysis/analysis/data/filenames.py

```diff
@@ -66,13 +71,19 @@
       import numpy as np
       import matplotlib.pyplot as plt
       from MDAnalysis.analysis.data.filenames import Janin_ref
-      X, Y = np.meshgrid(np.arange(-180, 180, 4), np.arange(-180, 180, 4))
+      X, Y = np.meshgrid(np.arange(-180, 180, 6), np.arange(-180, 180, 6))
       Z = np.load(Janin_ref)
       ax.contourf(X, Y, Z, levels=[1, 6, 600])
 
    The given levels will draw contours that contain 90% and 98% of the
    data. See the :ref:`Janin plot figure <figure-janin>` as an example.
```

Comment:
Same as above: change the text to reference the new figure.

Reply:
Please review the latest code push and let me know if it is resolved.